screens/HealthProfessionalScreen.js

```diff
@@ -1,158 +1,131 @@
-import React, { useState } from 'react';
-import { View, Text, TextInput, TouchableOpacity, Dimensions, StyleSheet, Keyboard } from 'react-native';
-
+import React, { useState }from 'react';
+import { View, Text, TextInput, TouchableOpacity, Image, Button, Dimensions, StyleSheet, Keyboard } from 'react-native';
+import MenuIcon from '../assets/images/menu-icon';
```

Comment:
Use this hp-menu-icon.svg instead. Since this is the health professional screen not patient screen. 

```suggestion
import MenuIcon from '../assets/images/hp-menu-icon.svg';
```